src/containers/CardDeck.js

```diff
@@ -0,0 +1,337 @@
+								{
+									"type": "text",
+									"text": "Thank You"
+								}
+							]
+						},
+						{
+							"type": "paragraph",
+							"attrs": {
+								"hidden": false
+							},
+							"content": [
+								{
+									"type": "text",
+									"marks": [
+										{
+											"type": "code_inline"
+										}
+									],
+									"text": "print(\"Hope you liked this top secret presentation\")"
+								}
+							]
+						},
+						{
+							"type": "paragraph",
+							"attrs": {
+								"hidden": false
+							},
+							"content": [
+								{
+									"type": "text",
+									"text": "Hope you liked this "
+								},
+								{
+									"type": "text",
+									"marks": [
+										{
+											"type": "strong"
+										}
+									],
+									"text": "top secret presentation"
+								}
+							]
+						}
+					]
+				},
+				"selection": {
+					"type": "text",
+					"anchor": 0,
+					"head": 0
+				}
+			}
+		]
+		const backJSON = [
+			{
+				"doc": {
+					"type": "doc",
+					"content": [
+						{
+							"type": "heading",
+							"attrs": {
+								"hidden": false,
+								"level": 1
+							},
+							"content": [
+								{
+									"type": "text",
+									"text": "Back of Card 1"
+								}
+							]
+						}
+					]
+				},
+				"selection": {
+					"type": "text",
+					"anchor": 0,
+					"head": 0
+				}
+			},
+			{
+				"doc": {
+					"type": "doc",
+					"content": [
+						{
+							"type": "heading",
+							"attrs": {
+								"hidden": false,
+								"level": 1
+							},
+							"content": [
+								{
+									"type": "text",
+									"text": "Back of Card 2"
+								}
+							]
+						},
+						{
+							"type": "paragraph",
+							"attrs": {
+								"hidden": false
+							},
+							"content": [
+								{
+									"type": "text",
+									"text": "Flippity flip "
+								},
+								{
+									"type": "text",
+									"marks": [
+										{
+											"type": "strong"
+										}
+									],
+									"text": "flip! "
+								}
+							]
+						}
+					]
+				},
+				"selection": {
+					"type": "text",
+					"anchor": 0,
+					"head": 0
+				}
+			}
+		];
+		const content = [
+			{"index": 0, "front": frontJSON[0]},
+			{"index": 1, "front": frontJSON[1], "back": backJSON[0]},
+			{"index": 2, "front": frontJSON[2], "back": backJSON[1]},
+			{"index": 3, "front": frontJSON[3]}
+		]
+		this.setState({content: content})
+	}
+	
+	render = () => {
+		if (this.state.content == null){
+			return (
+				<Container>
+					<Card/>
+				</Container>
+			);
+		}else{
+			console.log("Actual Card Deck")
+			const cards = [];
+			for (let i = 0; i < this.state.content.length; i+=2) {
+				cards.push(
+					<Row className="justify-content-md-center">
+						<Col lg="12" xl="6">
+							<Card content={this.state.content[i]}/>
+						</Col>
+						<Col lg="12" xl="6">
+							{
+								(i < this.state.content.length-1) && <Card content={this.state.content[i+1]}/>
```

Comment:
should this `i < length` check wrap the col instead of the card? or would that mess up the grid formatting?

Reply:
I put it inside the `Col`, so that when we have an odd number of cards, the last card is left justified. If it were outside, the card would be centered (we can change this later if this is what is desired).

Reply:
sg, i think the last one centered is fine as well, but justified makes sense